src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -508,6 +508,9 @@ def _evp_pkey_to_private_key(self, evp_pkey):
             self.openssl_assert(dh_cdata != self._ffi.NULL)
             dh_cdata = self._ffi.gc(dh_cdata, self._lib.DH_free)
             return _DHPrivateKey(self, dh_cdata, evp_pkey)
+        elif key_type == getattr(self._lib, "EVP_PKEY_X448", None):
+            # EVP_PKEY_X448 is not present in OpenSSL < 1.1.1
+            return _X448PrivateKey(self, evp_pkey)
```

Comment:
GC?

Reply:
Any evp_pkey passed to this function has already been gc'd